docs/overview/integrate/pool-setup.md

```diff
@@ -16,7 +16,7 @@ There are several different types of liquidity pools on Osmosis, each with uniqu
 - Supercharged Liquidity (Concentrated Liquidity) Pool, and
 - CosmWasm Pools (e.g., Transmuter and Alloyed Assets)
 
-Note: As of [Proposal 669](https://www.mintscan.io/osmosis/proposals/669), the Pool Creation Fee is 400 OSMO
+Note: As of [Proposal 699](According to https://www.mintscan.io/osmosis/proposals/699), the Pool Creation Fee is 100 USDC.
```

Comment:
I don't think you mean to include 'According to' in the URL portion? Wouldn't this break the link?

Reply:
Thanks for the catch, resolved